src/main/java/io/github/syst3ms/skriptparser/expressions/ExprContextValue.java

```diff
@@ -19,13 +19,12 @@
  * @author Mwexim
  */
 public class ExprContextValue implements Expression<Object> {
-
 	static {
 		Parser.getMainRegistration().addExpression(
 				ExprContextValue.class,
 				Object.class,
 				false,
-				"[the] [1:(past|previous)|2:(future|next)] context-<.+>"
+				"[the] [(1:(past|previous)|2:(future|next))] [context-]<.+>"
```

Comment:
I'd rather not have a `<.+>` pattern (which this is, remove the optional stuff), and besides, why were these parentheses added?

Reply:
1. I made it optional because we discussed this earlier. It provided issues back then, but I tested it and it seemed fine. I will remove them as I have another idea in my head for better context-value patterns. you are right.
2. It simply bugged. The syntax sugar you added is (I think) not able to handle the two different parse marks. Either way, I encountered issues with it and this is an easy, simple fix that still keeps the pattern somewhat elegant.